functions/utils/getApolloClient.ts

```diff
@@ -0,0 +1,20 @@
+import { ApolloClient, InMemoryCache } from '@apollo/client'
+const GRAPHQL_ENDPOINT = 'https://api.uniswap.org/v1/graphql'
+
+//TODO: Figure out how to make ApolloClient global variable
+export default new ApolloClient({
+  connectToDevTools: true,
+  uri: GRAPHQL_ENDPOINT,
+  headers: {
+    'Content-Type': 'application/json',
+    Origin: 'https://app.uniswap.org',
+    'User-Agent':
```

Comment:
Why is the User-Agent set to this value?

Reply:
No particular reason LOL. According to Andrew I just needed a user-agent, so I used the one he sent me and have been using it ever since